addons/binding/org.openhab.binding.dscalarm/src/main/java/org/openhab/binding/dscalarm/internal/discovery/DSCAlarmBridgeDiscovery.java

```diff
@@ -0,0 +1,166 @@
+        }
+    }
+
+    @Override
+    protected void stopBackgroundDiscovery() {
+        logger.debug("Stop DSC Alarm Bridge background discovery");
+        if (envisalinkBridgeDiscoveryJob != null && !envisalinkBridgeDiscoveryJob.isCancelled()) {
+        	envisalinkBridgeDiscoveryJob.cancel(true);
+        	envisalinkBridgeDiscoveryJob = null;
+        }
+        
+        if (it100BridgeDiscoveryJob != null && !it100BridgeDiscoveryJob.isCancelled()) {
+        	it100BridgeDiscoveryJob.cancel(true);
+        	it100BridgeDiscoveryJob = null;
+        }
+    }
+    
+	private Runnable envisalinkBridgeDiscoveryRunnable = new Runnable() {
+		@Override
+		public void run() {
+			envisalinkBridgeDiscovery.discoverBridge();
+		}
+	};
+
+	private Runnable it100BridgeDiscoveryRunnable = new Runnable() {
+		@Override
+		public void run() {
+			it100BridgeDiscovery.discoverBridge();
+		}
+	};
+
+	/**
+	 * Method to add an Envisalink Bridge to the Smarthome Inbox.
+	 * 
+	 * @param ipAddress
+	 */
+	public void addEnvisalinkBridge(String ipAddress) {
+		logger.debug("addBridge(): Adding new Envisalink Bridge on {} to Smarthome inbox",ipAddress);
+		
+		String bridgeID = ipAddress.replace('.', '_');
+		Map<String, Object> properties = new HashMap<>(0);
+		properties.put(EnvisalinkBridgeConfiguration.IP_ADDRESS, ipAddress);
+		
+		try {
+			ThingUID thingUID = new ThingUID(DSCAlarmBindingConstants.ENVISALINKBRIDGE_THING_TYPE, bridgeID);
+			
+			if (thingUID != null) {
+	
+				DiscoveryResult result = DiscoveryResultBuilder.create(thingUID).withProperties(properties) //.withProperty(EnvisalinkBridgeConfiguration.IP_ADDRESS, ipAddress)
+						.withLabel("EyezOn Envisalink Bridge - " + ipAddress).build();
+				thingDiscovered(result);
+				
+				logger.debug("addBridge(): '{}' was added to Smarthome inbox.",result.getThingUID());
+		     }
+		}catch(Exception e) {
+			logger.debug("addBridge(): Error: {}",e);
+		}
+	}
+
+	/**
+	 * Method to add an IT-100 Bridge to the Smarthome Inbox.
+	 * 
+	 * @param port
+	 */
+	public void addIT100Bridge(String port) {
+		logger.debug("addBridge(): Adding new IT-100 Bridge on {} to Smarthome inbox",port);
+
+		String bridgeID = "";
+		boolean containsChar = port.contains("/");
+		
+		if(containsChar) {
+			String[] parts = port.split("/");
+			bridgeID = parts[parts.length - 1].toUpperCase();
+		}else {
+			bridgeID = port.toUpperCase();			
+		}
+		
+		Map<String, Object> properties = new HashMap<>(0);
+		properties.put(IT100BridgeConfiguration.SERIAL_PORT, port);
+		
+		try {
+			ThingUID thingUID = new ThingUID(DSCAlarmBindingConstants.IT100BRIDGE_THING_TYPE, bridgeID);
+			
+			if (thingUID != null) {
+	
+				DiscoveryResult result = DiscoveryResultBuilder.create(thingUID).withProperties(properties)
+						.withLabel("DSC IT-100 Bridge - " + port).build();
+				thingDiscovered(result);
+				
+				logger.debug("addBridge(): '{}' was added to Smarthome inbox.",result.getThingUID());
```

Comment:
those things are logged by the framework already, so either remove it or reduce to TRACE.